app/agent/model.scala

```diff
@@ -28,12 +29,30 @@ trait IndexedItemWithStack extends IndexedItem {
 
 case class AWSAccount(accountNumber: Option[String], accountName: String)
 
+sealed trait AwsRegionType
+case object Global extends AwsRegionType
+case object Regional extends AwsRegionType
+
 abstract class CollectorSet[T](val resource:ResourceType, accounts: Accounts) extends Logging {
```

Comment:
Another option that might reduce boilerplate:
```suggestion
abstract class CollectorSet[T](val resource:ResourceType, accounts: Accounts, val awsRegionType: Option[AwsRegionType]) extends Logging {
```

Just another way of expressing the same thing, but would reduce repetition as the regionType would be passed in as a parameter. The scaladoc could then be moved to describe the CollectorSet.